bears/verilog/VerilogLintBear.py

```diff
@@ -0,0 +1,18 @@
+from coalib.bearlib.abstractions.Linter import linter
+
+
+@linter(executable='verilator',
+        use_stderr=True,
+        output_format='regex',
+        output_regex=r'%(?P<severity>Error|Warning):\s*'
+        r'(?P<filename>.*v):(?P<line>\d+):\s*'
+        r'(?P<message>.*)')
+class VerilogLintBear:
+    """
+    Check code for syntactical errors in Verilog code.
```

Comment:
add more info, we need better bear docs. Describe more about the exact capabilities. It can do more than syntax I hope?